x/profiles/types/expected_keepers.go

```diff
@@ -20,9 +21,19 @@ type RelationshipsKeeper interface {
 // ChannelKeeper defines the expected IBC channel keeper
 type ChannelKeeper interface {
 	GetChannel(ctx sdk.Context, srcPort, srcChan string) (channel channeltypes.Channel, found bool)
-	GetNextSequenceSend(ctx sdk.Context, portID, channelID string) (uint64, bool)
-	SendPacket(ctx sdk.Context, channelCap *capabilitytypes.Capability, packet ibcexported.PacketI) error
-	ChanCloseInit(ctx sdk.Context, portID, channelID string, chanCap *capabilitytypes.Capability) error
+
+	// SendPacket is called by a module in order to send an IBC packet on a channel.
+	// The packet sequence generated for the packet to be sent is returned. An error
+	// is returned if one occurs.
+	SendPacket(
+		ctx sdk.Context,
+		channelCap *capabilitytypes.Capability,
+		sourcePort string,
+		sourceChannel string,
+		timeoutHeight clienttypes.Height,
+		timeoutTimestamp uint64,
+		data []byte,
+	) (uint64, error)
```

Comment:
SendPacket handles sequence and extra stuffs now. In addition ChanCloseInit is removed since it is impossible to close the channel from `x/profiles` by user.

https://github.com/cosmos/ibc-go/blob/main/docs/migrations/v5-to-v6.md#ics04---sendpacket-api-change